src/main/java/org/broadinstitute/hellbender/tools/exome/ExomeReadCounts.java

```diff
@@ -150,6 +226,22 @@
     )
     protected Transform transform = Transform.RAW;
 
+    @Argument(
+            doc = "File containing the exons for analysis",
+            shortName = EXOME_FILE_SHORT_NAME,
+            fullName = EXOME_FILE_FULL_NAME,
+            optional = true
+    )
+    protected File exomeFile = null;
```

Comment:
I rather have all the info in onTraversalStart(). Otherwise I would need to change think quite a bit.